packages/@adobe/spectrum-css-temp/components/table/index.css

```diff
@@ -485,15 +485,15 @@ svg.spectrum-Table-sortedIcon {
   }
 }
 
-.resize-ew *,
```

Comment:
Doesn't this mean that if you hover over another element that has a different cursor, we no longer override it?

Reply:
that's what `!important` is for

Reply:
I guess if body isn't the full size of the document that could be problematic
this is already a problem when dealing with an iframe though...
I can't find anything to lock in the cursor style without one of these expensive ways, was hoping that setting PointerCapture might do it by forcing all events to be funneled through the capturing element, but it appears that doesn't affect CSS selectors

Reply:
o, I see what you mean, yeah, we still need these `*`
I've added them back in